docs/application/dotnet/guides/nui/widget.md

```diff
@@ -0,0 +1,254 @@
+# NUI Widget Application
+
+[NUIWidgetApplication](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.NUIWidgetApplication.html) helps you to create Tizen.NET widget applications, which are small versions of Tizen.NET applications on the home screen.
+You can launch these applications on the home screen when you select them in the **Add Widget** menu.
+
+**Figure: NUI widget application**
+
+![NUI widget application](./media/dali_widget_application.png)
+
+The main features of NUIWidgetApplication include:
+
+- Create widget applications
+
+  You can [create a NUI widget application](#create-a-nui-widget-application), which usually is composed of one process having the main loop. The NUIWidgetApplication framework enables you to create a widget instance that shares the same resources within the process. The widget application can also share data with other applications.
+
+- Manage the life-cycle
+
+  You can manage the life-cycle of widget instances through callback methods that are triggered as the instance states are changed.
+
+- Manage widget instances
+  
+  Whenever a widget instance is requested, the framework creates one.
+  You can manage the widget instances by updating or destroying them, or by retrieving information about them.
+
+- Create the widget UI
+
+  The widget application can create a user interface that appears on the home screen or viewer.
+
+    > **Note**
+    >
+    > The UI of widget application limits scroll actions to provide a better user experience. It is recommended that you design a UI layout within the given screen size.
+    >
+    > To draw the layout, you must use the [`Window`](https://developer.tizen.org/dev-guide/csapi/api/Tizen.NUI.Window.html) instance received from the `OnCreate` callback. If you create additional windows, the stack of widget application windows gets corrupted. This is because the platform handles the widget application window in a particular way.
+
+## Application and Viewer
+
+[`NUIWidgetApplication`](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.NUIWidgetApplication.html) makes diverse class instances whenever [`WidgetView`](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.WidgetView.html) requests for a widget instance.
+
+The widget instance has its own life-cycle similar to that of the widget application. However, the widget instance is an object created by the widget viewer application. Many widget instances can be running on a widget application process.
+
+>**Note**
+>
+>The case to use many widget instances in one widget application is known as multi-instance. In some devices, the multi-instance may not be supported. If a device does not support multi-instance, an error message is displayed.
```

Comment:
Global: As per the new guidelines for style, please use the updated Note tag.
Reference: https://github.com/Samsung/tizen-docs/blob/master/styleguide/custom-style.md

>[!Note]
>The case to use many widget instances in one widget application is known as multi-instance. In some devices, the multi-instance may not be supported. If a device does not support multi-instance, an error message is displayed.